peps/pep-0216.rst

```diff
@@ -3,17 +3,13 @@ Title: Docstring Format
 Author: Moshe Zadka <moshez@zadka.site.co.il>
 Status: Rejected
 Type: Informational
-Content-Type: text/x-rst
 Created: 31-Jul-2000
 Post-History:
 Superseded-By: 287
 
 
-Notice
-======
-
-This PEP is rejected by the author.  It has been superseded by PEP
-287.
+.. rejected::
+   This PEP is rejected by the author.  It has been superseded by :pep:`287`.
```

Comment:
If the "rejection" was by the author, it is in fact withdrawn, not rejected, particularly so as an information PEP superseded by another PEP. Therefore,

```suggestion
.. withdrawn::
   This PEP is rejected by the author.  It has been superseded by :pep:`287`.
```